### PR TITLE
[KYUUBI #1709] Introduce a IT module for Kyuubi and FlinkSQL Engine t…

### PR DESCRIPTION
**Pull request by yanghua**
…o decouple with kyuubi server

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Review by yaooqinn — APPROVED** [review]

**Comment by yanghua** [comment]
@pan3793 Double check?

File: kyuubi-integ-tests/kyuubi-flink-engine-test/pom.xml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    <parent>
+        <artifactId>kyuubi-integ-tests</artifactId>
+        <groupId>org.apache.kyuubi</groupId>
+        <version>1.5.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>kyuubi-flink-engine-test_2.12</artifactId>
+    <name>Kyuubi IT Tests : Flink SQL Engine</name>
```
Comment (yaooqinn) [inline kyuubi-integ-tests/kyuubi-flink-engine-test/pom.xml:31]: Kyuubi Project IT Flink SQL

File: kyuubi-integ-tests/kyuubi-flink-engine-test/pom.xml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>kyuubi-integ-tests</artifactId>
```
Comment (yaooqinn) [inline kyuubi-integ-tests/kyuubi-flink-engine-test/pom.xml:23]: kyuubi-integration-tests

File: integration-tests/kyuubi-flink-it/pom.xml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            <version>${project.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.kyuubi</groupId>
+            <artifactId>kyuubi-server_${scala.binary.version}</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
```
Comment (yaooqinn) [inline integration-tests/kyuubi-flink-it/pom.xml:47]:
```suggestion
```

**Review by pan3793 — APPROVED** [review]

**Review by andrewchoi5 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**